habitat-lab/habitat/datasets/rearrange/run_episode_generator.py

```diff
@@ -4,12 +4,17 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+import ctypes
 import os
 import os.path as osp
 import random
+import sys
 from dataclasses import dataclass, field
 from typing import TYPE_CHECKING, Any, List
 
+flags = sys.getdlopenflags()
```

Comment:
We really should have a global entry point hat ensures this gets gets triggered.